framework_tvb/tvb/core/entities/model/model_operation.py

```diff
@@ -253,6 +253,7 @@ class Operation(Base, Exportable):
     user_group = Column(String, default=None)
     range_values = Column(String, default=None)
     estimated_disk_size = Column(Integer)
+    hpc_job_url = Column(String, default=None)
```

Comment:
This new column could be ok...

I am wondering if we could not unify somehow the storage mechanism for HPC and Cluster launches.
For Cluster, we have table `OperationProcessIdentifier`. See its usage in `ClusterSchedulerClient`